Google.PowerShell/CloudResourceManager/GoogleCloudProjectCmdlets.cs

```diff
@@ -0,0 +1,133 @@
+﻿using Google.Apis.CloudResourceManager.v1;
+using Google.Apis.CloudResourceManager.v1.Data;
+using System.Collections;
+using System.Collections.Generic;
+using System.Management.Automation;
+
+namespace Google.PowerShell.CloudResourceManager
+{
+    /// <summary>
+    /// <para type="synopsis">
+    /// Retrieves one or more Google Cloud projects.
+    /// </para>
+    /// <para type="description">
+    /// Retrieves one or more Google Cloud projects. The cmdlet will return all available projects if no parameter
+    /// is used. If -Name, -Id or -Label is used, the cmdlets will return the projects that match the given arguments.
+    /// </para>
+    /// <example>
+    ///   <code>PS C:\> Get-GcProject</code>
+    ///   <para>This command gets all the available Google Cloud projects.</para>
+    /// </example>
+    /// <example>
+    ///   <code>PS C:\> Get-GcProject -Name "My Project"</code>
+    ///   <para>This command gets the project that has the name "My Project".</para>
+    /// </example>
+    /// <example>
+    ///   <code>PS C:\> Get-GcProject -Id "my-id"</code>
+    ///   <para>This command gets the project that has the Id "My Project".</para>
+    /// </example>
+    /// <example>
+    ///   <code>PS C:\> Get-GcProject -Label {"environment" = "test"}</code>
```

Comment:
Don't Hashtables need to start with `@{` ? Isn't what you have here a scriptBlock?